includes/controller/guest_login_controller.php

```diff
@@ -314,12 +314,12 @@ function guest_register() {
                   )),
                   div('row', array(
                   div('col-sm-8', array(
-                    form_select('native_lang', _("Native Language").' '. entry_required(), languages(), $native_lang, 'English')
+                    form_select('native_lang', _("Native Language").' '. entry_required(), languages())
                       ))
                   )),
                   div('row', array(
                   div('col-sm-8', array(
-                          form_multiselect('other_langs', _("Other Languages"), languages(), $other_langs, 'English')
```

Comment:
form_multiselect method has only 4 parameters, 6 are used.
the default tag has been removed as $other_langs is never initialized while showing the view